### PR TITLE
Week15

### PR DESCRIPTION
**Pull request by up2code3**

### CODE REVIEW

**Comment by up2code3** [comment]
HELLO! Everything seems

**Comment by up2code3** [comment]
Hello, Everything is there, I just realized I left some //notes in there that we're for me, but I meant to get rid of them before the push. Also , I was having trouble getting xss to clean , I do not think its actually doing it's job. I was also not able to test helmet in postman. But the csrf and the rate limit, I was able to test and are working.

File: models/User.js [hunk 1]
```diff
@@ -33,13 +33,13 @@ UserSchema.pre("save", async function () {
   this.password = await bcrypt.hash(this.password, salt);
 });
 
-UserSchema.methods.createJWT = function () {
```
Comment (jsuoz121) [inline models/User.js:36]: Nice :)

**Review by jsuoz121 — APPROVED** [review]
Nice job! This can be a difficult one

**Pull request merged**